publickey.js

```diff
@@ -11,7 +13,7 @@ const S3O_PUBLIC_KEY_URL = 'https://s3o.ft.com/publickey';
  * @return {function}       Returns a function that in turn returns either the public key as
  *                          as a string or a promise that resolves to the public key string.
  */
-module.exports = function (debug) {
+const poller = (debug) => {
 	let publicKey;
 
 	const flagsPoller = new Poller({
```

Comment:
not your error, but I suspect this was originaly copied from another library, 

rename to publicKeyPoller

Reply:
I've left this as is - it's in a `publicKey` file and exported as `poller` so it seems redundant.